docs/content/reference/types/archetypes/asset3d.md

```diff
@@ -23,15 +23,13 @@ A prepacked 3D asset (`.gltf`, `.glb`, `.obj`, `.stl`, etc.).
 
 snippet: asset3d_simple
 
-<center>
```

Comment:
Was it intentional to remove the centering? It tends to look nicer with it on imho

Reply:
Yes. We should do this kind of stuff via CSS on landing